habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/BuildDeploymentArtifactsMojo.java

```diff
@@ -149,4 +168,25 @@ protected void setUpPlaceholderFileAsMavenArtifact() {
         project.getArtifact().setFile(mavenArtifactFile);
     }
 
-}
+    private void cacheWheelFiles() {
+        PoetryCommandHelper poetryHelper = createPoetryCommandHelper();
+        try{
+            File wheelSourceDirectory = new File(String.format("%s/dist", project.getBasedir()));
```

Comment:
C: I believe the `dist` directory is configurable. We could pull the configuration parameter that currently exists on the Clean mojo up to abstract mojo and re-use it here.

Reply:
There is a configuration in the clean mojo but it's purpose is to identify where the clean plugin will delete artifacts. Doesn't appear to define where the are placed? Can you confirm? 

<img width="352" alt="image" src="https://github.com/TechnologyBrewery/habushu/assets/6691969/4d4d1183-c1a6-4dbd-80b7-4cd52b37de90">

Reply:
The actual configuration happens as part of Maven's functionality I believe, but may need a little research.  If you look at the `<build>` configuration of the POM in habushu-mixology you'll see we set `dist` with the `<directory>` tag. I'm assuming if you changed that value it would change it to a different location.